tox.ini

```diff
@@ -7,7 +7,6 @@ skip_missing_interpreters=True
 passenv =
      NUMBER_OF_PROCESSORS
      LOKY_MAX_CPU_COUNT
-usedevelop = True
```

Comment:
This would break pyinstaller (`ModuleNotFoundError` importing loky when running the pyinstaller command)... Even without any `os.chdir` in the pyinstaller test.